docs/extras/beta/index.md

```diff
@@ -0,0 +1,79 @@
+# OpenTrials beta
+
+Hello fans of evidence-based medicine and open data!
+
+We've recently launched the public beta of OpenTrials, after announcing the start of work on the platform on [February 10th, 2016](http://opentrials.net/2016/02/10/opentrials-technical-roadmap/).
+
+We are very excited about OpenTrials, and its potential role as part of the ecosystem to change the way clinical trials are conducted and reported on.
+
+In this post we'll look at the features that are available in the OpenTrials beta release, and what is in the pipeline on the road to a proper v1 release.
+
+## Features
+
+OpenTrials is designed as a modular platform, employing a set of loosely coupled components that handle distinct tasks from the collection of data from external sources, through to the exposing of data for consumption via the API or a UI.
+
+In the [announcement post](http://opentrials.net/2016/02/10/opentrials-technical-roadmap/) for the platform, we presented two diagrams that describe the architecture of the platform, and the data model that OpenTrials employs. The design presented then has not significantly changed, and you can see those diagrams below for convenience.
+
+<img src="images/architecture.png" width="600" style="text-align: center; display: block; margin: 0 auto;" title="OpenTrials Architecture Overview">
+
+<img src="images/model.png" width="600" style="text-align: center; display: block; margin: 0 auto;" title="OpenTrials Data Model Overview">
+
+What those diagrams do not reveal is the actual feature set exposed to users, so we'll highlight the major features here.
+
+### API
+
+The API is currently at **v1**. The base endpoint for the API is [here](http://api.opentrials.net/v1/), and interactive documentation for the API can be found [here](http://api.opentrials.net/v1/docs/).
```

Comment:
Although the route is `api.explorer.net/v1`, the API is still unstable (i.e. `v0`). Maybe I should change that to `api.explorer.net/v0` to avoid confusion.